libraries/rush-lib/src/logic/LookupByPath.ts

```diff
@@ -146,6 +170,23 @@ export class LookupByPath<TItem> {
     return this.findChildPathFromSegments(LookupByPath.iteratePathSegments(childPath, this.delimiter));
   }
 
+  /**
+   * Searches for the item associated with `childPath`, or the nearest ancestor of that path that
+   * has an associated item.
+   *
+   * @returns the found item, or `undefined` if no item was found
+   *
+   * @example
+   * ```ts
+   * const tree = new LookupByPath([['foo', 1], ['foo/bar', 2]]);
+   * tree.findChildPathAndIndex('foo/baz'); // returns { item: 1, index: 4 }
+   * tree.findChildPathAndIndex('foo/bar/baz'); // returns { item: 2, index: 8 }
+   * ```
+   */
+  public findChildPathAndIndex(childPath: string): IPrefixMatch<TItem> | undefined {
```

Comment:
```suggestion
  public findNearestChildPathAndIndex(childPath: string): IPrefixMatch<TItem> | undefined {
```

Reply:
Is this a request for a general renaming of the API surface, or just this specific new method?

Reply:
General renaming of the API surface. This is a new API, just feel that the name suggested fits what it does better than the name it has is all

Reply:
Not renaming the existing API for now since some external code is known to take a dependency.